styles/Splunk/Inclusivity.yml

```diff
@@ -8,7 +8,11 @@ action:
 swap:
   abnormal: atypical or not typical
   blacklist: deny list, reject, deny, or exclude
+  disable: deactivate, deselect, hide, inactive, turn off, unavailable
```

Comment:
@wordsmeanstuff Would 'unavailable' replace 'disabled' instead of 'disable' (adj instead of verb)?

Reply:
You are totally right! Thanks for your keen eye, Fabri.

Reply:
You're welcome! Feel free to commit the changes.